README.md

```diff
@@ -20,3 +20,17 @@ or
 ```
 
 Then navigate to `localhost:8181` in a browser.
+
+### Project Dashboard
+
+The Oppia Project Dashboard uses the web crypto API, which requires a TLS connection.
+To start an HTTPS simple server with a generated self-signed certificate, execute:
+
+```shell
+$ python3 start.py
+```
+
+Then navigate to `https://localhost:8181/project-dashboard` in a browser.  Note that
+you will need to explicitly permit the browser to accept this self-signed certificate.
+Chrome won't let you do that, but there's a workaround.  Just click anywhere on the
```

Comment:
Does the override instruction need to be done for each page load?

Reply:
Doesn't look like it.  I quit chrome, reopened, and navigated back to the site and it loaded without the unsafe warning page.

Reply:
Cool, sounds good.